config/jobs/kubernetes-sigs/cluster-api-provider-digitalocean/cluster-api-provider-digitalocean-presubmits.yaml

```diff
@@ -0,0 +1,49 @@
+presubmits:
+  kubernetes-sigs/cluster-api-provider-digitalocean:
+  - name: pull-cluster-api-provider-digitalocean-build
+    always_run: true
+    decorate: true
+    path_alias: sigs.k8s.io/cluster-api-provider-digitalocean
+    spec:
+      containers:
+      - image: golang:1.11
+        command:
+        - make
+        args:
+        - compile
+  - name: pull-cluster-api-provider-digitalocean-test
+    always_run: true
+    decorate: true
+    path_alias: sigs.k8s.io/cluster-api-provider-digitalocean
+    spec:
+      containers:
+      - image: golang:1.11
+        command:
+        - make
+        args:
+        - test-unit
+  - name: pull-cluster-api-provider-digitalocean-verify
+    always_run: true
+    decorate: true
+    path_alias: sigs.k8s.io/cluster-api-provider-digitalocean
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20181024-be2f242dd-master
+        command:
+        - make
+        args:
+        - verify
+  - name: pull-cluster-api-provider-digitalocean-lint
+    always_run: true
+    decorate: true
+    path_alias: sigs.k8s.io/cluster-api-provider-digitalocean
+    spec:
+      containers:
```

Comment:
sorry, that was wrong indent in my comment :-)

Reply:
@krzyzacy Fixed :)